test_scripts/CloudAppRPCs/006_SetCloudAppProperties_InvalidParams.lua

```diff
@@ -7,8 +7,8 @@
 --  1) Application sends a SetCloudAppProperties RPC request(with invalid param)
 --
 --  Expected:
---  1) SDL responds to mobile app with "ResultCode: INVALID_DATA,
---        success: false
+--  1) SDL responds to mobile app with "ResultCode: WARNINGS and success: true
+--  2) The invalid enum was filtered out from the message
```

Comment:
This is changing the purpose of the test. I would suggest that you just make one of the other parameters invalid instead (ex. `authToken = 123456789`)